index.js

```diff
@@ -53,6 +68,16 @@ module.exports = {
 				driver.takeScreenshot().then(function (screenImg) {
 					imageName = filename + iterationLabel + ".png";
 					imagePath = result.props.autoBaseDir + "/report/";
+
+					if (process.env.SCREENSHOT_URL) {
+						imagePath = path.normalize(result.props.autoBaseDir + "/" + process.env.SCREENSHOT_URL + '/');
+					}
+
+					var imageDir = path.dirname(path.normalize(imagePath + imageName));
+					if (!fs.existsSync(imageDir)) {
+						fs.mkdirRecursive(imageDir);
+					}
```

Comment:
this is needed to create the image directory if it does not exist
imageName can have directories in it, this is why we need this var